scripts/script_requirements.txt

```diff
@@ -0,0 +1,2 @@
+click
+PyGithub
```

Comment:
@cpennington @doctoryes we should standardize on whether to use [PyGitHub](https://pypi.python.org/pypi/PyGithub) or [github3](https://pypi.python.org/pypi/github3.py/1.0.0a4) for a python GitHub API wrapper.